styles.css

```diff
@@ -0,0 +1,325 @@
+    height: 3.125rem;
+    width: 100%;
+  }
+
+  .knowledge {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .lang {
+    background-color: #1c528a;
+    color: white;
+    height: 18.063rem;
+    display: flex;
+    flex-direction: column;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    line-height: 3;
+    font-size: 1.25rem;
+    padding-top: 2rem;
+    position: relative;
+  }
+
+  .fram {
+    background-color: #e2505c;
+    color: white;
+    height: 18.063rem;
+    display: flex;
+    flex-direction: column;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    line-height: 3;
+    font-size: 1.25rem;
+    padding-top: 2rem;
+    position: relative;
+  }
+
+  .skills {
+    background-color: #febf52;
+    color: white;
+    height: 18.063rem;
+    display: flex;
+    flex-direction: column;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    line-height: 4;
+    font-size: 1.125rem;
+    padding-top: 2rem;
+    position: relative;
+  }
+
+  .lang li {
+    font-weight: 600;
+    line-height: 1.5;
+    font-size: 1.125rem;
+    margin-left: 3.125rem;
+  }
+
+  .fram li {
+    font-weight: 600;
+    line-height: 1.5;
+    font-size: 1.125rem;
+    margin-left: 3.125rem;
+  }
+
+  .skills li {
+    font-weight: 600;
+    line-height: 1.5;
+    font-size: 1.125rem;
+    margin-left: 3.125rem;
+  }
+
+  #title {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 800;
+    font-size: 1.25rem;
+    color: white;
+    line-height: 3;
+    list-style: none;
+  }
+
+  .get-contact {
+    background-color: #f9f9fa;
+  }
+
+  #name {
+    width: 100%;
+    height: 3.25rem;
+    box-sizing: border-box;
+  }
+
+  #email {
+    width: 100%;
+    height: 3.25rem;
+    box-sizing: border-box;
+  }
+
+  #comment-box {
+    width: 100%;
+    height: 7rem;
+    box-sizing: border-box;
+  }
+
+  footer {
+    background-color: #1a2236;
+    height: 8.563rem;
+  }
+
+  .resume {
+    color: #92dcdc;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    text-align: center;
+    font-size: 16px;
+    line-height: 24px;
+    padding: 28px 0;
+  }
+
+  .font-links {
+    color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: x-large;
+  }
+
+  .image4 {
+    position: relative;
+  }
+
+  .image5 {
+    position: relative;
+    left: -2.55rem;
+  }
+
+  .get-contact div .Collaboration {
+    margin: 1rem 0;
+  }
+
+  .hashtag {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+  }
+
+  .rectangle18 {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    filter: opacity(50%);
+  }
+
+  .rectangle19 {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    filter: opacity(50%);
+  }
+
+  .bulb {
+    position: absolute;
+    left: 9rem;
+    top: 5rem;
+    width: 20rem;
+    height: 20rem;
+    filter: opacity(50%);
+  }
+
+  button:hover {
+    color: white;
+    background-color: silver;
+  }
```

Comment:
- [ ] Good job 👏 adding the `:hover` state as recommended by the previous reviewer but please make sure you're following the given [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=dDJDmCvIKfLOVPmu-0) when doing so.
- [ ] First of all I recommend you use a class name (like `.about-btn`)  instead of the generic `button` selector here as all buttons don't have the same styling
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/229103422-70b8334b-3d8a-41fb-af3d-680cd49882d6.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/229103069-f73712cd-fdf0-4981-b576-25265e1693b5.png)
- [ ] Also please make sure you add the `:active` (clicked) state as well as shown in the above design.